baseplate.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/reddit/baseplate.go/tracing"
 )
 
+const (
+	// DefaultStopDelay is the default StopDelay to be used in Serve.
+	DefaultStopDelay = time.Second
```

Comment:
I know it doesn't change anything but it is easier to read IMO
```suggestion
	DefaultStopDelay = 1 * time.Second
```

Reply:
+1